openshift-install-powervs

```diff
@@ -1333,6 +1355,17 @@ function process_varfile {
   done
 }
 
+#-------------------------------------------------------------------------
+# Flavor
+#-------------------------------------------------------------------------
+function validate_flavor {
+   [[ -z "$flavortype" ]] &&  error "The -flavor '$flavortype' Please specify an option with flavor."
+
+   flavor_list=($(ls automation/compute-vars/))
+   (for i in "${flavor_list[@]}"; do [[ "$i" == "$flavortype.tfvars" ]] && exit 0; done)  || error "The -flavor '$flavortype' is not valid. Please select either [small | medium | large]."
```

Comment:
Specify `flavor_list` instead of [small | medium | large].

Reply:
the error function generates issues if flavor_list array used `./openshift-install-powervs: line 133: exit: medium.tfvars: numeric argument required`

Reply:
The second arg to error function is error code. You need to append the list to the string and pass it to the function.

Reply:
Updated the file